types/src/account_config.rs

```diff
@@ -212,6 +212,14 @@ pub struct AccountEvent {
 }
 
 impl AccountEvent {
+    pub fn new(amount: u64, account: AccountAddress, metadata: Vec<u8>) -> Self {
```

Comment:
I would leave a todo here to illustrate that it should only be used for libra-client testing purpose